sdk/src/azure/storage/az_storage_blobs_blob_client.c

```diff
@@ -130,7 +130,6 @@ AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
 
 AZ_NODISCARD az_result az_storage_blobs_blob_upload(
     az_storage_blobs_blob_client* client,
-    az_context* context,
```

Comment:
nit: Ideally, such API and code changes would be done in a separate PR, given this PR was meant for just readme/docs, so that it could be reviewed in more detail.

Reply:
Also, I am surprised this didn't result in any changes to tests. Do we have unit tests for this?